dtests/src/resources/scripts/cdcConnector/createAndLoadFewTables.sql

```diff
@@ -19,30 +19,62 @@ CREATE EXTERNAL TABLE staging_BANK
     USING com.databricks.spark.csv OPTIONS (path '/export/shared/QA_DATA/TMG_Data_20G/BANK.dat', header 'true', inferSchema 'true', nullValue 'NULL', maxCharsPerColumn '4096');
```

Comment:
Parametrize the variables in sql script and pass them through bt or conf file at runtime.

Reply:
Done